src/vehicle_gui.cpp

```diff
@@ -3250,6 +3250,31 @@ bool VehicleClicked(const Vehicle *v)
 	return _thd.GetCallbackWnd()->OnVehicleSelect(v);
 }
 
+/**
+ * Dispatch a "vehicle group selected" event if any window waits for it.
+ * @param begin iterator to the start of the range of vehicles
+ * @param end iterator to the end of the range of vehicles
+ * @return did any window accept vehicle group selection?
```

Comment:
Should be `@return True if X, else false` or similar, I think

Reply:
I'm just copying the doc for the `bool VehicleClicked(const Vehicle *v)` overload just above this code.  We should either change none or both, and I went with none because I don't think we should make unrelated changes to the PR.